README.md

```diff
@@ -34,14 +34,23 @@ As a customer
 So that I can check if I want to order something
 I would like to see a list of dishes with prices
 
+Class menu
+dishes = array
+
+
 As a customer
 So that I can order the meal I want
 I would like to be able to select some number of several available dishes
 
+Class order_food
+order = array
+
 As a customer
 So that I can verify that my order is correct
 I would like to check that the total I have been given matches the sum of the various dishes in my order
 
+order.prices = total
+
 As a customer
 So that I am reassured that my order will be delivered on time
 I would like to receive a text such as "Thank you! Your order was placed and will be delivered before 18:52" after I have ordered
```

Comment:
Would be better to have a more thorough plan. So I can see where you were hoping to get to